gapis/api/vulkan/replay.go

```diff
@@ -944,6 +944,17 @@ func (a API) Replay(
 		return numInitialCmdWithoutOpt, nil
 	}
 
+	frameLoopEndCmdID := func(cmds []api.Cmd) api.CmdID {
+		lastCmdID := api.CmdID(0)
```

Comment:
is 0 the right default value here? DO we want len(cmds) -1 ?

Reply:
0 is correct here, as we want to update it to the last terminated cmd. If non is terminated,  and we set to len(cmds) -1, then it is incorrect.

Reply:
A terminated command is one that completed? Or one that was aborted half way through?

Reply:
A terminated command is one that completed. See gapis/api/cmd.go:69

Reply:
OK, logic makes sense to me now! "Terminated" is an unfortunately ambiguous name though :(